README.md

```diff
@@ -36,581 +34,22 @@ The following set of features makes Artipie unique among all others:
-
-## Multitenancy
-
-You may want to run Artipie for your company, which has a few teams.
-Each team may want to have its own repository. To do this, you create
-a global configuration file `/etc/artipie.yml`:
-
-```yaml
-meta:
-  layout: org
-  storage:
-    type: fs
-    path: /tmp/artipie/data/my-docker
-  credentials:
-    type: file
-    path: _credentials.yml
-```
-
-If the `type` is set to `file`, another YAML file is required in the storage, with
-a list of users who will be allowed to create repos
-(each `pass` is combination or either `plain` or `sha256` and a text):
-
-
-```yaml
-credentials:
-  jane:
-    pass: "plain:qwerty"
-  john:
-    pass: "sha256:xxxxxxxxxxxxxxxxxxxxxxx"
-```
-
-If the `type` is set to `env`, the following environment variables are expected:
-`ARTIPIE_USER_NAME` and `ARTIPIE_USER_PASS`. For example, you start
-Docker container with the `-e` option:
-
-```bash
-docker run -d -v /var/artipie:/var/artipie` -p 80:80 \
-  -e ARTIPIE_USER_NAME=artipie -e ARTIPIE_USER_PASS=qwerty \
-  artipie/artipie:latest
-```
-
-## Metrics
-
-You may enable some basic metrics collecting and periodic publishing to application log
-by adding `metrics` to `meta` section of global configuration file `/etc/artipie.yml`:
-
-```yaml
-meta:
-  metrics:
-    type: log
-```
-
-## How to contribute
-
-Fork the repository, make changes, and send us a
-[pull request](https://www.yegor256.com/2014/04/15/github-guidelines.html). We will review
-your changes and apply them to the `master` branch shortly, provided
-they don't violate our quality standards. To avoid frustration, before
-sending us your pull request please run full Maven build:
-
-```
-$ mvn clean install -Pqulice
-```
-
-To avoid build errors use Maven 3.2+.
-
-## How to run it locally
-
-To run Artipie server locally, build it with `mvn clean package -Passembly`
-and run with *(change port if needed)*:
-```java
-java -jar target/artipie-jar-with-dependencies.jar --config=example/artipie.yaml --port=8080
-```
-Example configuration uses `org` layout of Artipie with two level hierarchy,
-user `test` with password `123`, and `default` storage in `./example/storage` direcotry.
-To access the dashboard open `http://localhost/test` in your browser and enter user credentials.
-
-
-Thanks to [FreePik](https://www.freepik.com/free-photos-vectors/party) for the logo.
+<div align="center">
+  <h2>
+    <a href="http://artipie.com/docs">Documentation</a>
+    <span> | </span>
+    <a href="./examples">Examples</a>
+  </h2>
+</div>
+
+<p align="center">
+  <a href="https://github.com/artipie/artipie/blob/master/LICENSE.txt">
+    <img src="https://img.shields.io/badge/license-MIT-green.svg" />
+  </a>
+  <a href="https://hub.docker.com/r/artipie/artipie">
+    <img src="https://github.com/artipie/artipie/workflows/Docker%20Release/badge.svg" />
+  </a> 
+  <a href="https://hub.docker.com/r/artipie/artipie">
+    <img src="https://img.shields.io/docker/v/artipie/artipie?sort=date" />
+  </a>  
```

Comment:
@Sammers21 I'd suggest adding codecov badge as it partially verify our statement about high quality code

Reply:
@olenagerasimova, I decided to remove the badge because the proofs we create would not be counted in the metric.

Reply:
I also think that we should be very accurate with the badges and put only important ones.